### PR TITLE
payload: separate blivet.utils & blivet.arch call

### PR DESCRIPTION
**Pull request by jkonecny12**
First step in separating all the storage calls to one place.

### CODE REVIEW

File: pyanaconda/payload/__init__.py [hunk 1]
```diff
@@ -38,6 +38,7 @@
 from pyanaconda.image import opticalInstallMedia, verifyMedia, verify_valid_installtree
 from pyanaconda.core.util import ProxyString, ProxyStringError
 from pyanaconda.core.regexes import VERSION_DIGITS
+from pyanaconda.payload import storage as payload_storage
```
Comment (poncovka) [inline pyanaconda/payload/__init__.py]: Please, move the functions to `pyanaconda.payload.utils`, this is too confusing.

File: pyanaconda/payload/storage.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+import blivet.util
+import blivet.arch
+
+from pyanaconda.anaconda_loggers import get_module_logger
+from pyanaconda.payload.errors import PayloadSetupError
+
+log = get_module_logger(__name__)
+
+
+def get_mount_device(mountpoint):
```
Comment (poncovka) [inline pyanaconda/payload/storage.py]: Just a note, don't expect that these functions will be provided by the Storage module. It is a very basic functionality that doesn't require a device tree or any additional storage support.

File: pyanaconda/payload/__init__.py (hunk 1 above)
Comment (jkonecny12) [reply pyanaconda/payload/__init__.py]: Will do.

File: pyanaconda/payload/storage.py (hunk 2 above)
Comment (jkonecny12) [reply pyanaconda/payload/storage.py]: We can discuss this later. It is definitely not a priority now.

**Comment by jkonecny12** [comment]
**UPDATED:**
*  Moved to payload.utils instead of payload.storage

**Pull request merged**